src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,77 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.exceptions.RegistrationException;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final RegistrationService REGISTRATION_SERVICE = new RegistrationServiceImpl();
+    private static final StorageDao STORAGE_DAO = new StorageDaoImpl();
+    private static final String DEFAULT_NEW_USER_LOGIN = "newUser";
+    private static final String DEFAULT_PASSWORD = "123456";
+    private static final String INVALID_PASSWORD = "12345";
+    private static final User DEFAULT_USER = new User("defaultUser", DEFAULT_PASSWORD,
+            User.MIN_AGE);
+
+    @BeforeEach
+    void beforeEach() {
+        Storage.people.clear();
+        STORAGE_DAO.add(DEFAULT_USER);
+        DEFAULT_USER.setId(0L);
+    }
+
+    @Test
+    void validUserRegistration_Ok() {
```

Comment:
add the name of the method you are testing at the beginning. likewise in the following cases